Integrations/MimecastV2/MimecastV2.yml

```diff
@@ -1540,8 +1551,7 @@ script:
       type: String
   - arguments:
     - default: false
-      description: List of file hashes to check if they have been seen within
-        an account.
+      description: List of file hashes to check if they have been seen within an account.
```

Comment:
List of file hashes to check if they were seen in an account.

Reply:
done